phpseclib/Math/BigInteger/Engines/PHP/Base.php

```diff
@@ -80,7 +80,7 @@ public static function isValidEngine()
      * @param string $class
      * @return \phpseclib\Math\BigInteger\Engines\PHP
      */
-    protected static function powModHelper(PHP $x, PHP $e, PHP $n, $class)
+    public static function powModHelper(PHP $x, PHP $e, PHP $n, $class)
```

Comment:
Engines/PHP/Base extends Engines/PHP so visibility shouldn't be an issue. Indeed, if visibility were an issue then one would think that changing the visibility from protected to private wouldn't break any unit tests but it does:

https://travis-ci.org/terrafrost/phpseclib/jobs/261813738